src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1061,19 +1131,19 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         return builder->CreateCall(fn, {str});
     }
 
-    llvm::Value* lfortran_str_item(llvm::Value* str, llvm::Value* idx1)
+    llvm::Value* lfortran_str_copy(llvm::Value* str, llvm::Value* idx1, llvm::Value* idx2)
     {
-        std::string runtime_func_name = "_lfortran_str_item";
+        std::string runtime_func_name = "_lfortran_str_copy";
         llvm::Function *fn = module->getFunction(runtime_func_name);
         if (!fn) {
             llvm::FunctionType *function_type = llvm::FunctionType::get(
                     character_type, {
-                        character_type, llvm::Type::getInt32Ty(context)
+                        character_type, llvm::Type::getInt32Ty(context), llvm::Type::getInt32Ty(context)
                     }, false);
             fn = llvm::Function::Create(function_type,
                     llvm::Function::ExternalLinkage, runtime_func_name, *module);
         }
-        return builder->CreateCall(fn, {str, idx1});
+        return builder->CreateCall(fn, {str, idx1, idx2});
     }
```

Comment:
I think this is to be reverted. Please see https://github.com/lcompilers/lpython/commit/2f45421c06c2bee6a9918ae71ca0b88530ebd6fd.